ide/static/js/content.js

```diff
@@ -1299,9 +1339,11 @@ class Content extends React.Component {
           <div id="sidebar-scroll" className="col-md-12">
              <h5 className="sidebar-heading">ACTIONS</h5>
              <TopBar
+              isPublicSharing={this.state.isShared}
```

Comment:
There is one flag variable in state of content.js  where I maintain the type of model as isShared or not, can't we use that here because defining a new one causes necessity to maintain consistency in both. If not possible then please rename this variable name in props

Reply:
I haven’t defined this, this was already in the existing code. isShared is the flag variable which is there in content.js file right?

Reply:
I didn’t exactly understand, could you please elaborate?

Reply:
There is one boolean flag in content.js state variable if you are passing it down as props use the same name and don't define new ones if possible